README.md

```diff
@@ -65,7 +65,17 @@ This extension provides full Puppet Language support for [Visual Studio Code](ht
 
 ## Requirements
 
-You will need to have Puppet Agent installed in order to fully use this extension. You can find instructions and installation links here:
+You will need to have the [Puppet Development Kit (PDK)](https://puppet.com/docs/pdk/1.x/pdk.html) or [Puppet Agent](https://puppet.com/docs/puppet/4.10/about_agent.html) installed in order to fully use this extension.
+
+**Note:** PDK version 1.5.0 or higher is required, as that was the first version to include the Puppet gem.
```

Comment:
I don't think we need to go into the specifics.  Perhaps `You will need to have the [Puppet Development Kit (PDK)](https://puppet.com/docs/pdk/1.x/pdk.html) version 1.5.0 or higher ... `